metals/src/main/scala/scala/meta/metals/MetalsServices.scala

```diff
@@ -493,10 +503,11 @@ class MetalsServices(
 
 object MetalsServices extends LazyLogging {
   lazy val cacheDirectory: AbsolutePath = {
+    val cacheVersion = "0.1"
```

Comment:
Do we just bump this every time we change format?

Reply:
Correct, I moved this variable to `Mtags`, added a detailed docstring and changed the number to `"1"` so upgrades are a simple increment. No need for semantic versioning